package.json

```diff
@@ -2,19 +2,20 @@
   "name": "huron",
   "description": "An in-browser prototyping tool built on top of webpack and kss-node",
   "author": "Alley Interactive",
-  "version": "2.1.6-beta.0",
+  "version": "2.1.6",
   "repository": {
     "type": "git",
     "url": "https://github.com/alleyinteractive/huron"
   },
   "scripts": {
     "doc": "./node_modules/.bin/jsdoc src -r -c ./config/jsdoc.json",
     "help": "./node_modules/.bin/babel-node src/cli/huron.js -h",
-    "dev-cli": "webpack --config \"config/webpack-node.config.js\" -d --watch --progress --env.process=dev",
+    "dev-cli": "concurrently \"webpack --config config/webpack-node.config.js -d --watch --progress --env.process=dev\" \"jest --watch\"",
```

Comment:
you probably want jest to be running in a different window than webpack, since jest has it's own interactive view.